### PR TITLE
more volunteers

### PR DESCRIPTION
**Pull request by devhawk**

### CODE REVIEW

File: documentation/directory.md [hunk 1]
```diff
@@ -20,11 +20,16 @@ _Note: **@usernames** in bold denote the primary point of contact for each team.
...
 
 # [Projects](https://github.com/search?q=topic%3Aproject+org%3Ahelpfulengineering&type=Repositories)
 
 <!-- Ideally -->
-Each project keeps its own team directory [inside its main repository](https://github.com/search?q=topic%3Aproject+org%3Ahelpfulengineering&type=Repositories)
+Each project keeps its own team directory [inside its main repository](https://github.com/search?q=topic%3Aproject+org%3Ahelpfulengineering&type=Repositories).
+Known project team members listed below
+
+Reach4Help
+* [@Joseph Ashwin](https://helpfulengineering.slack.com/team/U010ZPFB38V)
```
Comment (0x2b3bfa0) [inline documentation/directory.md]: Could we move this entry to the table above?

Comment (dmangot) [reply documentation/directory.md]: @devhawk ^^

Comment (devhawk) [reply documentation/directory.md]: Can do but I want to indicate these are team members not project liaisons

Comment (devhawk) [reply documentation/directory.md]: I added a column for "Other Team Members"

**Review by dmangot — APPROVED** [review]

**Pull request merged**